charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml

```
@@ -165,7 +165,7 @@ spec:
         - "--"
 {{- end }}
         - "--mode"
-        - "kubernetes"
+        - "{{ .Values.mode | default "kubernetes" }}"
```

```suggestion
        - "{{ .Values.mode }}"
```
Having a `default` here is a bit redundant since we have the default value of `kubernetes` specified in the `values.yaml`. I think it'd be cleaner to remove it here and leave it in the `values.yaml` that way we don't have to maintain the 'default' value in 2 locations.

---

done!